backend/routes/user.routes.ts

```diff
@@ -0,0 +1,60 @@
+import express = require('express');
+import { User } from '../models/user.models';
+import { auth } from '../middleware/auth';
+
+export const user3Routes = express.Router();
+
+user3Routes.post('/register', async (req: any, res: any) => {
+  try {
+    const user = new User(req.body);
+    await user.save();
+    const token = await user.generateAuthToken();
+    res.status(201).send({user, token});
+  } catch (error) {
+    res.status(400).send(error);
```

Comment:
Is this really a bad request? Or is this a 500 server-side error?

Reply:
400 is a bad-request error.